src/box/lua/schema.lua

```diff
@@ -1619,6 +1620,42 @@ local function func_id_by_name(func_name, level)
 end
 box.internal.func_id_by_name = func_id_by_name -- for space.upgrade
 
+-- Normalize array of fields `fields`:
+-- - fields referenced as name are resolved to 0-based field number.
+-- - fields referenced as 1-based field number are converted to 0-based.
+local function normalize_fields(fields, format, name, level)
```

Comment:
I'd rename `name` to `what` to make it clear that it's used for error reporting.

Reply:
Fixed.